bloges56/DateSim
# Dialogue scene change

## Description
- Changes the dialogue to match the character you're interacting with in the arcade, also changes dialogue with Deon if you win his game

## How to Test
1. Run SceneManagerScene
2. Talk to Deon
3. Win the puzzle game 
4. Talk to Deon again to see if the dialogue changed
5. Talk to Remington to make sure it switches to his dialogue (he doesn't have a second dialogue yet)


## Checklist
- [x] Does the first interaction with Deon work?
- [x] Does winning the puzzle game change the dialogue to reflect that?
- [x] If you leave the puzzle game early is the dialogue still the same as the first interaction?
- [x] Does Remington's test dialogue work?
- [x] Do all the other objects still work?
